index.js

```diff
@@ -1,8 +1,5 @@
-// Copyright IBM Corp. 2018. All Rights Reserved.
-// Node module: loopback-connector-cos
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
 
-'use strict';
```

Comment:
Please preserve `'use strict'` and the copyright headers in all JavaScript files. The strict mode allows the V8 runtime to catch many developer errors early on, the copyright headers are required by our policies AFAIK.

Reply:
`index.js` has been removed. The other JavaScript files have the `use strict` and copyright headers added to them and so I am closing this